### ISSUE TITLE
builds fail on travis-jenkins

### ISSUE THREAD

**Issue by lenkis** [tarantool/tarantool-c#62]
my local "cmake ." fails with:

```
CMake Error at test/CMakeLists.txt:33 (add_custom_target):
  add_custom_target cannot create target "test" because another target with
  the same name already exists.  The existing target is a custom target
  created in source directory
  "/Users/e.shebunyaeva/work/tarantool-c/third_party/msgpuck/test".  See
  documentation for policy CMP0002 for more details.
```

**Comment by lenkis** [comment]
Error gone on my side after re-cloning the project. Builds ok locally. But still fails at travis-jenkins.

**Comment by Totktonada** [comment]
It seems, it was fixed in bde24fe5ee724e9cfbab922c086027e6c12c8838 by setting the [CMP0037](https://cmake.org/cmake/help/latest/policy/CMP0037.html) policy to `OLD`. It works.

However now it issues a deprecation warning. We can eliminate it for cmake 3.11+ just like we did it in Tarantool: https://github.com/tarantool/tarantool/commit/a753f258f208f5070e5516a7162ed071e5948b3c.
